### PR TITLE
fix(CoA/EoE/Naxx): Several bug fixes

### LINKED ISSUE #37: <4445> Loatheb(25)

2 min "Next Inevitable Doom" timer still starts on pull, should be 30 seconds not 2 min
<4445> Loatheb(25)

2 min "Next Inevitable Doom" timer still starts on pull, should be 30 seconds not 2 min  - raised by Holycoke via Discord link: https://discord.com/channels/808813760978223135/882719072763121685/884793098440413214

### LINKED ISSUE #40: <4445> Gluth(25) 

enrage timer is wrong, boss enrage after 4th Decimate, not on the current timer..

hack fix would be to increase the enrage timer by 1 min, should at least make it MORE accurate
<4445> Gluth(25) 

enrage timer is wrong, boss enrage after 4th Decimate, not on the current timer..

hack fix would be to increase the enrage timer by 1 min, should at least make it MORE accurate  - raised by Holycoke via Discord link: https://discord.com/channels/808813760978223135/882719072763121685/884825118872854568

### LINKED ISSUE #41: <4445> Thaddius(25)
Next Magnetic pull timer needs a 0.5 second reduction

this timer is on a loop correct?
<4445> Thaddius(25)
Next Magnetic pull timer needs a 0.5 second reduction

this timer is on a loop correct?  - raised by Holycoke via Discord link: https://discord.com/channels/808813760978223135/882719072763121685/884832993083011195

### LINKED ISSUE #49: <4448> Malygos(10)

Reduce enrage timer by 15 seconds
<4448> Malygos(10)

Reduce enrage timer by 15 seconds  - raised by Holycoke via Discord link: https://discord.com/channels/808813760978223135/882719072763121685/885496164781277204

### LINKED ISSUE #50: <4448> Sartharion(10)
add 12 min enrage timer on pull
<4448> Sartharion(10)
add 12 min enrage timer on pull  - raised by Holycoke via Discord link: https://discord.com/channels/808813760978223135/882719072763121685/885503428367884299

### LINKED ISSUE #51: <4448> Sartharion(10)
3D
"Tenebron arrives" timer needs a 3 second increase
<4448> Sartharion(10)
3D
"Tenebron arrives" timer needs a 3 second increase  - raised by Holycoke via Discord link: https://discord.com/channels/808813760978223135/882719072763121685/885503617178669098

### LINKED ISSUE #52: <4448> Sartharion(10)
3D
Vesperon and Shadron arrive timers need to be swapped
<4448> Sartharion(10)
3D
Vesperon and Shadron arrive timers need to be swapped  - raised by Holycoke via Discord link: https://discord.com/channels/808813760978223135/882719072763121685/885505780386791425

### PR DESCRIPTION
**Pull request by dnzr**
closes #37 
closes #40
closes #41 
closes #49 
closes #50 
closes #51 
closes #52

first try at doing anything like this, let me know if its BS 👍

### CODE REVIEW

**Comment by Exeasd** [comment]
Well, its mostly a good job, but we dont actually change revisions by ourselves. Ace setup auto release action that does it for us, so its usually best to just keep your fork up to date and not to make changes to core :)

File: DBM-ChamberOfAspects/Obsidian/Sartharion.lua [hunk 1]
```diff
@@ -29,9 +29,10 @@ mod:AddBoolOption("AnnounceFails", true, "announce")
 
 local timerShadowFissure    = mod:NewCastTimer(5, 59128)--Cast timer until Void Blast. it's what happens when shadow fissure explodes.
 local timerWall             = mod:NewCDTimer(30, 43113)
-local timerTenebron         = mod:NewTimer(30, "TimerTenebron", 61248)
-local timerShadron          = mod:NewTimer(80, "TimerShadron", 58105)
-local timerVesperon         = mod:NewTimer(120, "TimerVesperon", 61251)
+local timerTenebron         = mod:NewTimer(33, "TimerTenebron", 61248)
+local timerShadron          = mod:NewTimer(120, "TimerShadron", 58105)
+local timerVesperon         = mod:NewTimer(80, "TimerVesperon", 61251)
+local enrageTimer			= mod:NewBerserkTimer(720)
```
Comment (Exeasd) [inline DBM-ChamberOfAspects/Obsidian/Sartharion.lua:36]: For those drake timers to actually take effect You need to change we handle it in function CheckDrakes

File: DBM-Core/DBM-Core.lua [hunk 2]
```diff
@@ -41,7 +41,7 @@
 -------------------------------
 --  Globals/Default Options  --
 -------------------------------
-DBM_REVISION = "$Revision: 4449 $"
+DBM_REVISION = "$Revision: 4450 $"
```
Comment (Exeasd) [inline DBM-Core/DBM-Core.lua:45]: And well, this revision has to be brought back to 4449 or we will skip revision.

**Review by Siarkowy — APPROVED** [review]
Great job man! Thanks for contributing.

Unfortunately, due to current release process, gotta file all the changes under a single patch note.

Let's keep it this way now but for better visibility, I would ask for smaller, multiple PRs (by boss or by instance). Thanks!!

**Pull request merged**